### PR TITLE
add init version of revpi-bluetooth

### PR DESCRIPTION
**Pull request by zhan-kunbus**

### CODE REVIEW

**Comment by iluminat23** [comment]
Why is the branch called main?

**Comment by zhan-kunbus** [comment]
> Why is the branch called main?

main is now by default in github, should we always change it to master?

File: usr/bin/revpi-btuart [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  OPTS=""
+  TYPEID="any"
+  SPEED=""
+  FLOW=""
+  BDADDR=""
+fi
+
+if [ ! -d "$DEVPATH" ]; then
+  # ttyREVPIBT is symlink created in revpi-tools/50-revpi.rules
+  $HCIATTACH $OPTS ttyREVPIBT $TYPEID $SPEED $FLOW $BDADDR
```
Comment (iluminat23) [inline usr/bin/revpi-btuart:34]: I don't see any 50-revpi.rules in revpi-tools. And I don't like to have it in there. Creating symlinks everywhere is pretty messy. For the RS485 it is ok, as it is user visible. But no user needs to access the Bluetooth uart at any point.

File: usr/bin/revpi-btuart [hunk 2]
```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+# referenced pi-bluetooth, bluetooth attach for Revolution Pi BT
+
+revpi_dev_type()
+{
+  return 0; # flat
+  return 1; # cm4
+  return 2; # cm4 with bluetooth
```
Comment (iluminat23) [inline usr/bin/revpi-btuart:8]: This is incomplete code?

**Review by iluminat23 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**